src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -239,40 +239,6 @@ public static void Namespace_Property_Set_Invalid_Value_Throws(string value)
         Assert.Throws<ArgumentNullException>(() => binding.Namespace = value);
     }
 
-    public static MemberDataSet<Encoding> ValidEncodings
```

Comment:
Reviewers: this section was removed because they were already in TestData.cs.  I wanted them to exist in only one place, and in the one place that had an rd.xml for its types.